prow/cmd/jenkins-operator/README.md

```diff
@@ -144,7 +144,7 @@ adding comments about failed tests. Note that this functionality may
 potentially move into its own service, then the Jenkins operator will
 not need to contact the Github API. The required options are already
 defaulted:
-* `github-token-file` set to `/etc/github/oauth`. This is the Github bot
+* `github-token-path` set to `/etc/github/oauth`. This is the Github bot
```

Comment:
Is this a user-facing change?

Reply:
> Is this a user-facing change?

Yes, but it's not breaking because [this][1].  Folks will need to update at some point before we drop the fallback.

[1]: https://github.com/kubernetes/test-infra/pull/9503/commits/8e4a1c4eacf6f4e9fe3c47a5e2f75727b53ac2d7#diff-10d11420786ce54a3c335b47d1a5e063R43